projects/canaryprotocol/index.js

```diff
@@ -0,0 +1,24 @@
+const { VAULTS } = require("./constants")
+
+// test with
+// node test.js projects/canaryprotocol/index.js
```

Comment:
The output is
```
--- plume ---
pETH                      2.35103741
pUSD                      1.1098734600000002
USDT                      0.1
Total: 3.5609108700000003
```
I assume your price engine does not support nRA nYield and nTBILL? Cause we have a few cents TVL on these assets as well